packs/equipment-effects/effect-dragonfly-potion.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Dragonfly Potion",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dragonfly Potion]</p>\n<p>You gain a fly Speed equal to your land Speed.<p></p>Additionally, you gain low-light vision and a +2 item bonus to visual Perception checks, and you can't be flanked except by creatures higher level than you are (though lower-level creatures can still help their higher-level allies flank).</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dragonfly Potion]</p>\n<p>You gain a fly Speed equal to your land Speed. Additionally, you gain low-light vision and a +2 item bonus to visual Perception checks.</p>"
```

Comment:
It might be possible to handle the flanking denial. Take a look at "Deny Advantage".